plugins/catalog-backend-module-gitlab/src/providers/GitlabOrgDiscoveryEntityProvider.test.ts

```diff
@@ -260,15 +260,16 @@ describe('GitlabOrgDiscoveryEntityProvider - refresh', () => {
     const taskDef = schedule.getTasks()[0];
 
     await (taskDef.fn as () => Promise<void>)();
-    const userEntities = mock.expected_full_org_scan_entities.filter(
+    const entities = mock.expected_full_org_scan_entities.filter(
```

Comment:
Does this logic change the default behaviour which we have today or am I misreading this test change?

Reply:
The full org scan returns users as well as groups (and their subgroups).
Previously, there was no testing with subgroups. I implemented this into the test data. So the comparison of returned users and the returned entities doesn't make a lot of sense anymore.

I can make the test more precise by filtering the returned entities to users, compare this to the expected users and add another line of test that filters the returned entities to groups and compares them to the expected number of groups. That would keep the original test logic in place but still would be compatible to the test data.